cypress/plugins/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+module.exports = (on, config) => {
```

Comment:
What is the purpose of this plugin?

Reply:
It creates logs for failed tests which are displayed on commandline and also save these logs in json format in the logs folder